### PR TITLE
docs: Explain how to add new Grafana dashboard and Prometheus ServiceMonitor

### PR DESCRIPTION
**Pull request by alexandre-allard**

### CODE REVIEW

File: docs/operation/cluster_and_service_configuration.rst [hunk 1]
```diff
@@ -247,6 +247,40 @@ applied with Salt.
...
+     name: <grafana-dashboard-name>
+     namespace: metalk8s-monitoring
+   data:
+     <dashboard-filename>.json: |-
+       <dashboard-definition>
+
+.. note::
+
+   The `grafana_dashboard: '1'` label in the example above
+   is mandatory for the dashboard to be taken into account.
```
Comment (TeddyAndrieux) [inline docs/operation/cluster_and_service_configuration.rst]: Maybe mention that it must be in "metalk8s-monitoring" namespace also

Comment (alexandre-allard) [reply docs/operation/cluster_and_service_configuration.rst]: Yes, I put it in the definition above, but better be explicit.

**Review by TeddyAndrieux — APPROVED** [review]

**Pull request merged**